src/blocks/stopwatch.scss

```
@@ -0,0 +1,43 @@
+.stopwatch {
+  position: relative;
+  @include circleSize($size-stop-watch);
+
+  background: $bg-stop-watch;
+  
+  &__minut::before {
+    content: "";
+    @include positArrow($width-minut);
```

Fix the typos here and all over the project.
```suggestion
    @include positArrow($width-minute);
```